tests/frontend/specs/helper.js

```diff
@@ -136,7 +136,7 @@ describe("the test helper", function(){
       helper.selectLines($startLine, $endLine, startOffset, endOffset);
 
       var selection = inner$.document.getSelection();
-      expect(cleanText(selection.toString())).to.be("ort lines to t");
+      expect(cleanText(selection.toString().replace(/(\r\n|\n|\r)/gm,""))).to.be("ort lines to t");
```

Comment:
This is logically part of the next commit where you do the same in other places, right?

I'll move it there and comment from that point.

Reply:
To be clear, I am referring to this: 6a7bcfdb6552f06829bcdba5ace3fca484d0a38a

Reply:
Yea.

Reply:
I'm terrible at committing, I drank too much of the "commit often" juice.. 💯   Sorry about that!  At least I show my work tho, I like to show how dumb I can be / how my brain goes about solving / sorting things so when I have to fix something (in this case 6 years later?) I can see what I was trying to do and why I did something weird.

I do a lot of weird.  But that's okay 🍡

Reply:
No problem. I'm just trying to make sure we won't get lost in another 6 years... :)